src/OrleansAWSUtils/Storage/Provider/DynamoDBStorageProvider.cs

```diff
@@ -0,0 +1,340 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Newtonsoft.Json;
+using System.Threading;
+using Orleans.Serialization;
+using Amazon.DynamoDBv2.Model;
+using Amazon.DynamoDBv2;
+using OrleansAWSUtils;
+using System.IO;
+using OrleansAWSUtils.Storage;
+
+namespace Orleans.Storage
+{
+    public class DynamoDBStorageProvider : IStorageProvider
```

Comment:
When someone other than you tries to figure out your thought process when implementing this, they won't go to the Orleans documentation for details. Please add a short summary like I mentioned, I agree that a more inclusive and exhaustive documentation should be added to Orleans documentation. Also, having public entities without a summary raises compilation warnings in Orleans.